borg/cache.py

```diff
@@ -154,6 +151,25 @@ def _do_open(self):
         except configparser.NoSectionError:
             self.close()
             raise Exception('%s does not look like a Borg cache.' % config_path) from None
+        # borg < 1.0.8rc1 had different canonicalization for the repo location (see #1655 and #1741).
```

Comment:
this is now done independent of the cache version.
it could be also just done only for cache version 1.